administration/config/webpack.config.ts

```diff
@@ -512,10 +392,26 @@ const createWebpackConfig = (webpackEnv: 'development' | 'production'): webpack.
         ],
       }),
     ].filter(Boolean),
-    // Turn off performance processing because we utilize
-    // our own hints via the FileSizeReporter
-    performance: false,
+    devServer: {
+      compress: true,
+      host: '0.0.0.0', // This enables devices in the same network to connect to the dev server
+      hot: true,
+      historyApiFallback: true,
+      port: 3000,
+      static: {
+        directory: paths.appPublic,
+      },
+    },
```

Comment:
:thought_balloon:  You can add `open: true,` to devServer to make it open automatically like it was.

Reply:
Done :)